specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/2017-04-18/cognitiveservices.json

```diff
@@ -785,12 +785,14 @@
           "enum": [
             "Creating",
             "ResolvingDNS",
+            "Moving",
+            "Deleting",
             "Succeeded",
             "Failed"
           ],
           "x-ms-enum": {
             "name": "ProvisioningState",
-            "modelAsString": false
+            "modelAsString": true
```

Comment:
This is going to cause breaking changes in the SDKs.

Reply:
Yes, so I will update the SDK (C# version). I changed it to string as we cannot capture all possible state. Currently because it is enum the SDK and PS throws an exception on unexpected state.